custom/icds_reports/management/commands/rebuild_ccs_record_ucr.py

```diff
@@ -9,31 +9,41 @@
 
 
 class Command(BaseCommand):
-    help = "Rebuild ccs record cases ucr for pnc_complete"
+    help = "Rebuild open ccs record cases and their corresponding person cases "
 
-    def handle(self):
-        indicator_config_id = 'static-ccs_record_cases'
-        case_type_of_xmlns = 'ccs_record'
+    def get_asdfsd(self, indicator_config_id, case_type_of_xmlns):
```

Comment:
does `asdfsd` stand for something or is this just some garbage name? (if the latter probably makes sense to change)

Reply:
argh!! Super sorry. made a random function for quickly clean the code. I will change now.